tests/e2e/server/wait.mjs

```diff
@@ -0,0 +1,6 @@
+const delay = 0;
+export default async function wait() {
```

Comment:
Just wanted a quick way to mimic latency, if desired. This might be better written as a middleware function.